src/azure-cli-core/setup.py

```diff
@@ -72,7 +72,9 @@
     'six',
     'tabulate>=0.7.7',
     'wheel',
-    'azure-mgmt-resource==2.1.0'
+    'azure-mgmt-resource==2.1.0',
+    'pyperclip==1.7.0',
+    'psutil==5.6.1'
```

Comment:
Looks like this dependency is causing our ADO check to fail. I suspect that this dependency isn't playing nice inside of alpine.